msg/AudioNote.msg

```diff
@@ -0,0 +1,3 @@
+# Play a note (frequency) for a specific duration
+uint16_t frequency    # Frequency of note in Hz.
```

Comment:
There is no _t in the ros2 message datatype specification, it should just be `uint16`
https://docs.ros.org/en/foxy/Concepts/About-ROS-Interfaces.html#field-types

Reply:
You also need to add your messages to the CMakeLists.txt in the `set(msg_files`